legal-api/src/legal_api/models/business.py

```diff
@@ -283,6 +284,14 @@ class AssociationTypes(Enum):
     batch_processing = db.relationship('BatchProcessing', lazy='dynamic')
     jurisdictions = db.relationship('Jurisdiction', lazy='dynamic')
 
+    @dataclass
+    class AffiliationSearchDetails:  # pylint: disable=too-many-instance-attributes
+        """Used for filtering Identifiers based on filters passed."""
```

Comment:
might be easier if you simplify these the class is already a "search"

similar to how you did in auth